example/reduce/src/alpakaConfig.hpp

```diff
@@ -45,10 +45,10 @@ static constexpr uint64_t getMaxBlockSize()
 //! \tparam TAcc The accelerator type.
 //!
 //! Defines the appropriate iterator for an accelerator.
-template<typename T, typename TBuf, typename TAcc>
+template<typename T, typename TBuf, typename TMemoryHandle, typename Idx, typename TAcc>
```

Comment:
Another good opportunity for `BufferAccessor` IMO.

Reply:
I think I disagree. We should still use the normal `Accessor` in all places that are generic/library code. And the reduction implementation here looks like it is an attempt of a generic implementation. Why not allow it on non-buffer accessors?